Feat: [OR-191] Added Banner on Installation page (#37)
gsoft-inc/wl-orbiter: docs/getting-started/Installation.stories.mdx

# Installation

> <Banner>If you are working with Sharegate, we recommend using <ExternalLink href="https://orbit.sharegate.design/">Orbit</ExternalLink>.</Banner>

Reviewer: ```suggestion
<Banner>If you are working with Sharegate, you should use <ExternalLink href="https://orbit.sharegate.design/">Orbit</ExternalLink>.</Banner>
```
